.gitlab-ci.yml

```
@@ -93,7 +93,18 @@ test-linux-stable:
     # but still want to have debug assertions.
     RUSTFLAGS: "-Cdebug-assertions=y -Dwarnings"
   script:
-    - time cargo test --all --release --locked -- --include-ignored
+    - time cargo nextest run --all --release --locked --run-ignored all
+
+test-doc:
+  stage:                           test
+  <<:                              *docker-env
+  <<:                              *common-refs
+  variables:
+    # Enable debug assertions since we are running optimized builds for testing
+    # but still want to have debug assertions.
+    RUSTFLAGS: "-Cdebug-assertions=y -Dwarnings"
+  script:
+    - time cargo test --doc
```

```suggestion
    - time cargo test --doc --no-deps
```

we don't really care of testing the deps here, do we?